src/Command/Ide/IdeListMineCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Acquia\Cli\Command\Ide;
+
+use AcquiaCloudApi\Endpoints\Applications;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class IdeListMineCommand.
+ */
+class IdeListMineCommand extends IdeCommandBase {
+
+  protected static $defaultName = 'ide:list-mine';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('List available Cloud IDEs belonging to you');
```

Comment:
```suggestion
    $this->setDescription('List Cloud IDEs belonging to you');
```

Normally when we talk about "available" Cloud IDEs, we're referring to available entitlements. Not sure if customers see it the same way, but could be confusing.